### PR TITLE
add spot instances to senza doc

### PR DESCRIPTION
**Pull request by szuecs**

### CODE REVIEW

File: components/senza.rst [hunk 1]
```diff
@@ -564,3 +582,5 @@ Another use case for cross-stack references if one needs to access outputs from
 .. _jq: https://stedolan.github.io/jq/
 .. _"ELB" health check type for the auto scaling group: http://docs.aws.amazon.com/AutoScaling/latest/DeveloperGuide/healthcheck.html
 .. _valid AWS Cloud Formation ELB properties: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-elb.html
+.. _AWS spot instance: https://aws.amazon.com/de/ec2/spot/
+.. _AWS instance prices: http://www.ec2instances.info/?region=eu-central-1
```
Comment (hjacobs) [inline components/senza.rst]: can you remove the region here?

File: components/senza.rst [hunk 2]
```diff
@@ -477,6 +477,24 @@ This component supports the following configuration properties:
 ``VolumeSize``
     How Much GB should this EBS have?
 
+**SpotPrice**
```
Comment (hjacobs) [inline components/senza.rst:480]: can you also add the property below `InstanceType` in the definition list above?

File: components/senza.rst (hunk 1 above)
Comment (szuecs) [reply components/senza.rst]: done

File: components/senza.rst (hunk 2 above)
Comment (szuecs) [reply components/senza.rst:480]: I hope I understood you correctly

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**